bitcoinetl/streaming/streaming_utils.py

```diff
@@ -1,16 +1,46 @@
 from blockchainetl.jobs.exporters.console_item_exporter import ConsoleItemExporter
 
 
-def get_item_exporter(output):
-    if output is not None:
-        from blockchainetl.jobs.exporters.google_pubsub_item_exporter import GooglePubSubItemExporter
-        item_exporter = GooglePubSubItemExporter(
-            item_type_to_topic_mapping={
-                'block': output + '.blocks',
-                'transaction': output + '.transactions'
-            },
-            message_attributes=('item_id',))
-    else:
-        item_exporter = ConsoleItemExporter()
+def get_item_exporter(output,topic_mapping):
 
+    item_exporter_type = determine_item_exporter_type(output)
+    if item_exporter_type == ItemExporterType.PUBSUB:
+        if output is not None:
+            from blockchainetl.jobs.exporters.google_pubsub_item_exporter import GooglePubSubItemExporter
+            item_exporter = GooglePubSubItemExporter(
+                item_type_to_topic_mapping={
+                    'block': output + '.blocks',
+                    'transaction': output + '.transactions'
+                },
+                message_attributes=('item_id',))
+        else:
+            item_exporter = ConsoleItemExporter()
+    
+    elif item_exporter_type == ItemExporterType.KAFKA:
+        from blockchainetl.jobs.exporters.kafka_exporter import KafkaItemExporter
+        if (topic_mapping is None):
+            item_exporter = KafkaItemExporter(output, item_type_to_topic_mapping={
+                'block': 'producer.bitcoin.hot.blocks',
+                'transaction': 'producer.bitcoin.hot.transactions',
+            })
```

Comment:
change topic names to 
producer-bitcoin-blocks-hot
producer-bitcoin-transactions-hot